.github/workflows/deploy.yaml

```diff
@@ -40,24 +64,31 @@ jobs:
         run: |
           docker push ${IMAGE}
 
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    if: always() && (needs.build.result == 'success' || needs.build.result == 'skipped')
```

Comment:
Hva er hensikten med `always()` her?

Reply:
`always()` er nødvendig for å kunne kjøre `deploy`-jobben selv om build-job er skipped. se https://github.com/actions/runner/issues/491.

Grunnen til det er at GHA setter inn [`success()`](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#success)-funksjonen i if-testen, dersom ingen `job status check function`(funksjonene oppgitt [her](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions)) er oppgitt. `success()` returnerer false hvis tidligere job ble `skipped`, og deploy ville derfor ikke blitt kjørt.